src/Compilers/Test/Utilities/CSharp/FunctionPointerUtilities.cs

```diff
@@ -128,6 +130,42 @@ static void verifyParameter(ParameterSymbol symbol, MethodSymbol containing)
             }
         }
 
+        public static void VerifyPublicFunctionPointerGetTypeInfo(SemanticModel model, SyntaxNode syntax, bool useConvertedType,
```

Comment:
>syntax [](start = 98, length = 6)

I find this loosely typed helper extremely confusing and hard to follow at use-sites.